src/KeyStreamCapture/KeyStreamCapture.csproj

```diff
@@ -11,7 +11,8 @@
     <RootNamespace>KeyStreamCapture</RootNamespace>
     <AssemblyName>KeyStreamCapture</AssemblyName>
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <TargetFrameworkProfile>Client</TargetFrameworkProfile>
```

Comment:
Everything else is .net 4.0 and I had compilation issues because this was client profile referencing full framework stuff.